test/metabase/test/data/generic_sql.clj

```diff
@@ -271,19 +319,23 @@
 
 (def DefaultsMixin
   "Default implementations for methods marked *Optional* in `IGenericSQLTestExtensions`."
-  {:add-fk-sql                default-add-fk-sql
-   :create-db-sql             default-create-db-sql
-   :create-table-sql          default-create-table-sql
-   :database->spec            default-database->spec
-   :drop-db-if-exists-sql     default-drop-db-if-exists-sql
-   :drop-table-if-exists-sql  default-drop-table-if-exists-sql
-   :execute-sql!              default-execute-sql!
-   :load-data!                load-data-chunked!
-   :pk-field-name             (constantly "id")
-   :prepare-identifier        (u/drop-first-arg identity)
-   :qualified-name-components default-qualified-name-components
-   :qualify+quote-name        default-qualify+quote-name
-   :quote-name                default-quote-name})
+  {:add-fk-sql                    default-add-fk-sql
+   :inline-column-comment-sql     (constantly "")
+   :standalone-column-comment-sql (constantly "")
+   :inline-table-comment-sql      (constantly "")
+   :standalone-table-comment-sql  (constantly "")
```

Comment:
probably better just to make these all be `(constantly nil)` rather than blank strings and then have the function that actually uses their values check to see if it gets something back before using the value

Reply:
otherwise someone could implement this function for a database, and if it were to return `nil` it would cause a `NullPointerException` in the call to `format` where you actually use it